src/test/java/org/apache/accumulo/access/grammar/antlr/AccessExpressionAntlrEvaluator.java

```diff
@@ -0,0 +1,119 @@
+package org.apache.accumulo.access.grammar.antlr;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.antlr.v4.runtime.tree.TerminalNode;
+import org.apache.accumulo.access.AccessEvaluator;
+import org.apache.accumulo.access.AccessExpression;
+import org.apache.accumulo.access.Authorizations;
+import org.apache.accumulo.access.IllegalAccessExpressionException;
+import org.apache.accumulo.access.grammars.AccessExpressionParser.Access_expressionContext;
+import org.apache.accumulo.access.grammars.AccessExpressionParser.Access_tokenContext;
+import org.apache.accumulo.access.grammars.AccessExpressionParser.And_expressionContext;
+import org.apache.accumulo.access.grammars.AccessExpressionParser.And_operatorContext;
+import org.apache.accumulo.access.grammars.AccessExpressionParser.Or_expressionContext;
+import org.apache.accumulo.access.grammars.AccessExpressionParser.Or_operatorContext;
+
+public class AccessExpressionAntlrEvaluator implements AccessEvaluator {
+
+  private class Entity {
+
+    private Set<String> authorizations;
+
+    @Override
+    public String toString() {
+      return "Entity [authorizations=" + authorizations + "]";
+    }
+
+  }
+
+  private final List<Entity> entities;
+
+  public AccessExpressionAntlrEvaluator(List<Authorizations> authSets) {
+    entities = new ArrayList<>(authSets.size());
+
+    for (Authorizations a : authSets) {
+      Set<String> entityAuths = a.asSet();
+      Entity e = new Entity();
+      entities.add(e);
+      e.authorizations = new HashSet<>(entityAuths.size() * 2);
+      a.asSet().stream().forEach(auth -> {
+        e.authorizations.add(auth);
+        e.authorizations.add(new String(AccessEvaluator.escape(auth.getBytes(UTF_8), true), UTF_8));
```

Comment:
Can use this existing API method.

```suggestion
        var quoted = AccessExpression.quote(auth);
        if (!quoted.startsWith("\"")) {
          quoted = '"' + quoted + '"';
        }
        e.authorizations.add(quoted);
```

Reply:
Implemented change in 6fd8db5